modules/process/pages/uipath.adoc

```diff
@@ -11,7 +11,7 @@ All connectors share common authentication settings, and have other specific set
 
 [CAUTION]
 ====
-Versions 2.3.0 and later of the connector require a Bonita Studio released after October 2025. +
+Versions 2.3.0 and later of the connector require a Bonita Studio released after October 2025. In particular, starting from 2025.1-u1. +
```

Comment:
Not sure to get it.
Maybe we can list the versions ? Versions 2024.3-u5, 2024.2-u7, 2024.1-u9, 2023.2-u11 or higher